README.md

```diff
@@ -27,37 +27,55 @@ var clientsManager = RelationsGrpcClientsManager.forInsecureClients(url);
 var checkClient = clientsManager.getCheckClient();
 ```
 ### Making requests
-Synchronous:
+#### Synchronous example
+
+Let's say we want to check whether a subject of type "user" with id "bob" has "view" permission to a resource of type "workspace" and id "hosts". We can make the following synchronous check request:
+
 ```java
 var checkRequest = CheckRequest.newBuilder()
                 .setSubject(SubjectReference.newBuilder()
-                                .setObject(ObjectReference.newBuilder()
-                                        .setType("user")
-                                        .setId("joe").build())
-                                .build())
+                        .setSubject(ObjectReference.newBuilder()
+                                .setType(ObjectType.newBuilder()
+                                        .setName("user").build())
```

Comment:
I copied it from `example/Caller.java`, but I don't run the main() method (against my local relations-api) with every commit, so I didn't catch the error. Client side validation and a test is really what's needed, because running the above manually against a running relations-api before every PR is a PITA. Good call.